requirements.txt

```diff
@@ -74,3 +74,5 @@ uuid==1.30
 watchdog==0.6.0
 xmltodict==0.9.2
 yolk==0.4.1
+
+git+https://github.com/mccalluc/django_docker_engine.git@v0.0.1
```

Comment:
This hits a redirect: It lives at https://github.com/refinery-platform/django_docker_engine now.